SteamReview/SteamReview/Steam.cpp

```diff
@@ -185,24 +205,24 @@ FGame FSteam::AddNewGame()
 	std::cout << "Please enter Day ";
```

Comment:
This is a bit vague, what day? any day? Mind elaborating a bit, same for the month and year messages